packages/@webex/internal-plugin-metrics/src/prelogin-metrics-batcher.ts

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable valid-jsdoc */
```

Comment:
Why are these here? It appears that all of your JSDoc is invalid currently, or, the ESLint rule needs to be updated.

`./.eslintrc.js`, within the `.ts` overrides:

![Screenshot 2024-03-14 at 10 21 04 AM](https://github.com/webex/webex-js-sdk/assets/14828820/bbb30df9-5c33-4ce7-985e-ef28dffe5a22)

Note that this updates some of the error messages (they wont be correct due to our extremely damaged eslint/typescript configuration), but when these changes exist, it works as intended.

![Screenshot 2024-03-14 at 10 20 28 AM](https://github.com/webex/webex-js-sdk/assets/14828820/31d7729a-9f48-4dd6-9248-2ac48b61fd1d)

Reply:
I just copied this over from the other batcher file.

Reply:
I've fixed all of these, in prelogin and call diagnostic batcher

Reply:
Fixed the batcher.js too